pmm-app/src/react-plugins-deps/components/Table/Table.tsx

```diff
@@ -0,0 +1,115 @@
+// @ts-nocheck
+
+import React, { FC, ReactElement } from 'react';
+import { useRowSelect, useTable } from 'react-table';
+import { Spinner, useTheme } from '@grafana/ui';
+import { getStyles } from './Table.styles';
+
+interface TableInterface {
+  columns: object[];
+  data: object[];
+  noData?: ReactElement;
+  ActionPanel?: FC<{ selected: any[] }>;
+  loading?: boolean;
+  rowKey?: Function;
+}
+function Table({ columns, data, ActionPanel, noData, loading, rowKey }: TableInterface) {
+  const theme = useTheme();
+  const Styling = getStyles(theme);
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow, selectedFlatRows } = useTable(
+    {
+      columns,
+      data,
+    },
+    useRowSelect,
+    hooks => {
+      if (ActionPanel) {
+        hooks.visibleColumns.push(columns => [
+          {
+            id: 'selection',
+            Header: ({ getToggleAllRowsSelectedProps }) => (
+              <div data-qa="select-all">
+                <label className="checkbox-container checkbox-container--main no-gap">
```

Comment:
These classes produce a lot of noise. We should either use Grafana's Checkboxes or, if those don't fit the purpose, our own styled components.

Reply:
moved the checkbox to a temporarily separate component, will be rewritten as part of the table implementation part